x/gamm/module.go

```diff
@@ -66,7 +66,7 @@ func (AppModuleBasic) ValidateGenesis(cdc codec.JSONCodec, config client.TxEncod
 	if err := cdc.UnmarshalJSON(bz, &genState); err != nil {
 		return fmt.Errorf("failed to unmarshal %s genesis state: %w", types.ModuleName, err)
 	}
-	return genState.Validate()
+	return nil
```

Comment:
wat

Reply:
Ah we removed the genesis validation, because right now it left most params unvalidated, it only checked params.

Reply:
That's correct. The only validation that used to be done is the fee param that was moved to swap router